tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/LabelHelper.cs

```diff
@@ -57,6 +58,34 @@ public static ServiceLabelStatus CheckServiceLabel(string csvContent, string ser
             return ServiceLabelStatus.DoesNotExist;
         }
 
+        public static bool CheckServiceLabelInReview(IReadOnlyList<Octokit.PullRequest?> pullRequests, string serviceLabel)
+        {
+            try
```

Comment:
Because the service label is normalized into the name of the branch, that can be used to detect a service label PR.